### PR TITLE
Update license date 🗝 ..

### PR DESCRIPTION
**Pull request by 3imed-jaberi**

### CODE REVIEW

File: src/Skeleton.elm [hunk 1]
```diff
@@ -94,7 +94,7 @@ footer =
     , a [ class "grey-link", href "https://github.com/elm/compiler/" ] [ text "Compiler Source" ]
     , text " — "
     , a [ class "grey-link", href "https://github.com/elm/elm-lang.org/" ] [ text "Site Source" ]
-    , text " — © 2012-2019 Evan Czaplicki"
+    , text " — © 2012-2020 Evan Czaplicki"
```
Comment (3imed-jaberi) [inline src/Skeleton.elm]:
```suggestion
    , text " — © 2012-present Evan Czaplicki"
```

**Comment by 3imed-jaberi** [comment]
cc @evancz !!

**Comment by avh4** [comment]
@3imed-jaberi please do not @-mention or request reviewers on an unsolicited PR to a public project.  It will be reviewed when the maintainers get to it.

**Comment by 3imed-jaberi** [comment]
> @3imed-jaberi please do not @-mention or request reviewers on an unsolicited PR to a public project. It will be reviewed when the maintainers get to it.

@avh4 , it's an urgent pr... just a small word (as I whisper) the unwanted word suggests confusion (bad) .. According to your words, all the pr are unsolicited. !! (we are in 2020 😊)

be more kind and thanks ❤️ !

File: src/Skeleton.elm (hunk 1 above)
Comment (mabotn) [reply src/Skeleton.elm]: I think "Present" is better than "present"

**Comment by 3imed-jaberi** [comment]
@evancz, @elm/all ...

All that you had as managers of this repo was to say that we don't welcome any contribution.
The ELM project did not succeed .. Here I saw the reason for this ..
You have the most bad maintainers. 

remember be kind ^^ ! ..

**Pull request closed without merging**